spec/lib/constants_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+RSpec.describe Constants do
```

Comment:
i'm not sure it's necessary to have a spec test on the _data_ for the constants. imho the existence and value type of the constant is fine for the spec. otherwise you have to maintain the data in two places.